frontend/src/hooks/useKeycloakWrapper.tsx

```diff
@@ -12,27 +19,42 @@ function useKeycloakWrapper() {
   const hasClaim = (claim?: string) => {
     return !claim || keycloak?.hasResourceRole(claim);
   };
+  const userInfo = keycloak?.userInfo as IUserInfo;
```

Comment:
@devinleighsmith @goemen @asanchezr Please review this merged file.  I'm not certain it's correct.